### ISSUE TITLE
Umowa własna z kurierem DPD, jak ją poprawnie wskazać przy wysłaniu powiadomienia na order/checkout-forms/{id}/shipments

### ISSUE THREAD

**Issue by grzechuum** [allegro/allegro-api#6113]
Posiadamy w panelu (zamówienia > ustawienia > umowy własne) zdefiniowaną umowę własną z kurierem DPD o nazwie "umowa własna z DPD". Etykiety DPD generujemy za pomocą API DPD i nie korzystamy z api [parcel-management/parcel-create-commands](https://developer.allegro.pl/documentation/#operation/createNewParcel).

Metoda /parcel-management/delivery-services zwraca mi rekord z umową własną:

[9] => Array
                (
                    [id] => 19439771
                    [service] => DPD
                    [name] => umowa własna z DPD
                    [owner] => CLIENT
                    [carrierId] => DPD 
...

Jak prawidłowo wskazać umowę własną przy potwierdzeniu nr przesyłki za pomocą metody order/checkout-forms/{id}/shipments ?
Podajemy tutaj carrierId oraz carrierName ale carrierName jest brany pod uwagę tylko dla carrierID='OTHER'. Jakie powinny tutaj być wartości dla carrierId i carrierName  aby powiadomić  Allegro o wygenerowanym numerze przesyłki z umową własną?

**Comment by MaciejFrackowiak** [comment]
Dla wskazanej metody dostawy przyporządkowany jest "carrierId"=DPD. Dla zasobu [POST /order/checkout-forms/{id](https://developer.allegro.pl/documentation/#operation/createOrderShipmentsUsingPOST)}/shipments wymaganą wartością jest carrierId, czyli w tym przypadku "DPD". Wartość dla pola "carrierName" może być podana, gdy tylko jako "carrierdId" wskażesz OTHER.

**Comment by grzechuum** [comment]
Ok to czy ustawienie dla 
[POST /order/checkout-forms/{id}/shipments](https://developer.allegro.pl/documentation/#operation/createOrderShipmentsUsingPOST)

carrierdId=OTHER
carrierName='umowa własna z DPD'

poprawnie przypisze umowę własną do nr przesyłki?

**Comment by MaciejFrackowiak** [comment]
Nie, właściwą wartością jest DPD zgodnie z polem "carrierId".

**Comment by grzechuum** [comment]
ok, czyli wskazanie umowy własnej jest tak naprawdę niemożliwe przez ten interfejs i dostępne jedynie poprzez 
[parcel-management/parcel-create-commands](https://developer.allegro.pl/documentation/#operation/createNewParcel)
?

**Comment by MaciejFrackowiak** [comment]
Tak, umowa własna nie ma żadnego znaczenia dla przekazania identyfikatora przewoźnika przy trackingu przesyłki.

**Comment by grzechuum** [comment]
ślicznie dziękuję za info
